src/TableFunctions/TableFunctionS3.h

```diff
@@ -26,7 +26,8 @@ class TableFunctionS3 : public ITableFunction
                                       " - url, format, structure, compression_method\n"
                                       " - url, access_key_id, secret_access_key, format\n"
                                       " - url, access_key_id, secret_access_key, format, structure\n"
-                                      " - url, access_key_id, secret_access_key, format, structure, compression_method\n"
+                                      " - url, access_key_id, secret_access_key, format, structure, compression_method\n",
+                                      " - url, access_key_id, secret_access_key, session_token, format, structure, compression_method\n"
```

Comment:
I think we should also support signature `url, access_key_id, secret_access_key, session_token, format`